adapter/svcctrl/google_service_control.go

```diff
@@ -0,0 +1,115 @@
+package svcctrl
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/pborman/uuid"
+
+	sc "google.golang.org/api/servicecontrol/v1"
+
+	"istio.io/mixer/adapter/svcctrl/config"
+	"istio.io/mixer/pkg/adapter"
+	"istio.io/mixer/template/metric"
+)
+
+type builder struct {
+	createClientFn
+}
+
+type handler struct {
+	serviceControlClient *sc.Service
+	env                  adapter.Env
+	configParams         *config.Params
+}
+
+func (b *builder) Build(c adapter.Config, env adapter.Env) (adapter.Handler, error) {
+	client, err := b.createClientFn(env.Logger())
+	if err == nil {
+		return nil, err
+	}
+	params := c.(*config.Params)
+	handler := &handler{
```

Comment:
Just do

   return &handler{

No need for a temp var.

Reply:
done